pkg/obs/nodeserver.go

```diff
@@ -87,7 +88,8 @@ func (ns *nodeServer) NodePublishVolume(_ context.Context, req *csi.NodePublishV
 	if err := ns.Mount.MakeDir(targetPath); err != nil {
 		return nil, status.Errorf(codes.Internal, "Failed to make dir: %s, error: %v", targetPath, err)
 	}
-	if err := createCredentialFile(ns.Driver.cloud.Global.AccessKey, ns.Driver.cloud.Global.SecretKey); err != nil {
+	credentialFile, err := createCredentialFile(ns.Driver.cloud.Global.AccessKey, ns.Driver.cloud.Global.SecretKey)
```

Comment:
It is best to delete `credentialFile` file after the fun finishes

Reply:
done. add "defer deleteCredentialFile" function in nodeserver